packages/react/src/themes/teams/components/Flex/flexItemVariables.ts

```diff
@@ -0,0 +1,15 @@
+import { pxToRem } from '../../../../lib'
+
+import { FlexItemSize } from '../../../../components/Flex/FlexItem'
+
+type SizeValues = { [Key in FlexItemSize]: string }
+
+export type FlexItemVariables = SizeValues
+
+export default (): FlexItemVariables => ({
+  'size.half': '50%',
+  'size.quarter': '25%',
+
+  'size.small': pxToRem(150),
```

Comment:
how did we come up with these values?
can we make these part of theme somehow or is that plan for future?

Reply:
it is definitely a plan for the future - there are no clear specs on that yet, provided numbers are free to be changed